### PR TITLE
Feature/migrate old workspaces to subscriptions

### PR DESCRIPTION
**Pull request by geclos**

### CODE REVIEW

**Review by andresgutgon — APPROVED** [review]

File: packages/core/drizzle/0068_thin_mandroid.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-- Identify all workspaces that do not have a current subscription assigned to
+-- them and insert a new subscription for each of these workspaces
+WITH new_subscriptions AS (
+    INSERT INTO subscriptions (workspace_id, plan, created_at, updated_at)
+    SELECT w.id, 'hobby_v1', NOW(), NOW()
+    FROM workspaces w
+    LEFT JOIN subscriptions s ON w.id = s.workspace_id
+    WHERE w.current_subscription_id IS NULL
+    RETURNING id, workspace_id
+)
```
[the remark this answers is not in the document]
Comment (geclos) [reply packages/core/drizzle/0068_thin_mandroid.sql:13]: yes subscriptions there's:

1 workspace - n subscriptions

and also:

1 workspace - 1 current subscription

[the remark this answers is not in the document]
Comment (csansoon) [reply packages/core/drizzle/0068_thin_mandroid.sql:13]: What does it mean when a workspace has multiple subscriptions?

[the remark this answers is not in the document]
Comment (andresgutgon) [reply packages/core/drizzle/0068_thin_mandroid.sql:13]: The rest of subscriptions are old subscriptions. Current susbcription is workspace.current_subscription_id

[the remark this answers is not in the document]
Comment (geclos) [reply packages/core/drizzle/0068_thin_mandroid.sql:13]: that ☝🏼

**Pull request merged**